File: python/PalindromeNumber.py
# Runtime: 56 ms
# Memory Usage: 14.2 MB

class Solution:
    def isPalindrome(self, x: int) -> bool:
        x = str(x)

        for i in range(int(len(x)/2)):
            if(x[i] != x[len(x)-1-i]):
                return False
        return True

File: java/PalindromeNumber.java
// Runtime: 19 ms
// Memory Usage: 41.8 MB

class Solution {
    public boolean isPalindrome(int x) { // 12221

        String input = Integer.toString(x);
        int length = input.length();

        for (int i = 0; i <= length / 2; i++) {
            if (input.charAt(i) != input.charAt(length - 1 - i)) {
                return false;
            }
        }
        return true;
    }
}